src/mapnik_vector_tile.cpp

```diff
@@ -2063,17 +1984,28 @@ NAN_METHOD(VectorTile::toGeoJSON)
         NanReturnUndefined();
     }
 
-    handle_to_geojson_args(layer_id,
-                           tiledata,
-                           closure->all_array,
-                           closure->all_flattened,
-                           error_msg,
-                           closure->layer_idx);
-    if (!error_msg.empty())
+    try
+    {
+        vector_tile::Tile tiledata = detail::get_tile(closure->v->buffer_);
+
+        handle_to_geojson_args(layer_id,
+                               tiledata,
+                               closure->all_array,
+                               closure->all_flattened,
+                               error_msg,
+                               closure->layer_idx);
+        if (!error_msg.empty())
+        {
+            delete closure;
```

Comment:
No coverage of this line, this one of those ones that really sucks to get at as well, but just noting, also exception below is not covered.